docs/pages/reference/backends.mdx

```diff
@@ -556,6 +560,9 @@ teleport:
     write_min_capacity: int
     write_max_capacity: int
     write_target_value: float
+    # Enables On-Demand billing
+    # default: true
+    on_demand: [true|false]
```

Comment:
I would move this above the read/write capacity values and also mention here that the capacity values are ignored for on-demand mode.